src/components/BBanner/BBanner.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <header role="banner" :class="classes">
+    <slot></slot>
+  </header>
+</template>
+
+<script>
+import BNav from '../BNav'
+
+export default {
+  name: 'b-banner',
+  props: {
+    primary: Boolean,
+    secondary: Boolean,
+    tirtiary: Boolean
```

Comment:
tertiary

Reply:
VSCode says it has spell check. Lies.

Reply:
Are these exclusive booleans like only one can be true? Is there a way we can enforce that?

Reply:
You can't check other props. You can do that in the computed class function.

Reply:
Yeah we should do that because presumably it's not valid for it to have all 3 classes